src/condor_ce_jobmetrics

```diff
@@ -33,27 +33,33 @@ def secure_json_write(fname, contents):
             json.dump(contents, temp_file)
             os.rename(temp_fname, fname)
         except:
-            print "Error writing json file"
+            print >> sys.stderr, "Error writing json file"
     finally:
         temp_file.close()
 
+
+def _newstat():
+    return {"Running": 0, "Idle": 0, "Held": 0, "Jobs": 0}
+
 def process_one_schedd(ad):
     import htcondor
     import classad
     ad = classad.ClassAd(ad)
     schedd = htcondor.Schedd(ad)
     status_map = {1: "Idle", 2: "Running", 5: "Held"}
-    _newstat = lambda: {"Running": 0, "Idle": 0, "Held": 0, "Jobs": 0}
     total_results = _newstat()
+    gpu_total_results = _newstat()
     vo_results = collections.defaultdict(_newstat)
     job_count = collections.defaultdict(_newstat)
-    print "Processing CE %s." % ad.get("Name", "Unknown")
+    gpu_job_count = collections.defaultdict(_newstat)
+    print >> sys.stderr, "Processing CE %s." % ad.get("Name", "Unknown")
     try:
         for job in schedd.xquery("true", ["JobStatus", 'x509UserProxyVOName',
```

Comment:
```suggestion
        for job in schedd.xquery("RoutedJob is UNDEFINED", ["JobStatus", 'x509UserProxyVOName',
```

Let's exclude routed jobs. Feel free to fix-up the whitespace for the `schedd.xquery()` call as you see fit!